src/main/java/io/ncbpfluffybear/supserv/SupServItems.java

```diff
@@ -72,4 +72,12 @@ private SupServItems() {}
     public static final SlimefunItemStack DRAGON_EGG = new SlimefunItemStack("DRAGON_EGG",
         Material.DRAGON_EGG, null);
 
+    public static final SlimefunItemStack FARMERS_HOE = new SlimefunItemStack("FARMERS_HOE",
+            Material.IRON_HOE,
+            "&3Farmers&8Hoe",
+            "",
+            "&7Makes sure harvested plots get reseeded, at the cost of durability",
```

Comment:
Split this into two lines, the lore currently gets forced out of the screen for me
![image](https://user-images.githubusercontent.com/31554056/121469022-61943880-c981-11eb-9390-d610078a22e8.png)